chainservice/builder.go

```diff
@@ -344,6 +348,15 @@ func (builder *Builder) buildContractStakingIndexer(forTest bool) error {
 		return err
 	}
 	builder.cs.contractStakingIndexer = indexer
+	// indexer v2
```

Comment:
in a separate function, because of https://github.com/iotexproject/iotex-core/pull/4305/files#diff-c65dd5a657c73eb89458c69a82a25fc0993655650b3ad477136955c31e50e2bbR327-R333

Reply:
okay